src/components/data-exploration/heatmap/HeatmapMetadataTrackSettings.jsx

```diff
@@ -12,18 +12,19 @@ import {
 
 import { updatePlotConfig } from '../../../redux/actions/componentConfig';
 import ReorderableList from '../../ReorderableList';
+import { getCellSets } from '../../../redux/selectors';
 
 const HeatmapMetadataTrackSettings = (props) => {
   const dispatch = useDispatch();
 
   const { componentType } = props;
 
-  const cellSets = useSelector((state) => state.cellSets);
+  const cellSets = useSelector(getCellSets());
   const selectedTracks = useSelector(
     (state) => state.componentConfig[componentType].config.selectedTracks,
   );
 
-  const getCellSets = (category) => {
+  const getCellSetsHierarchy = (category) => {
```

Comment:
You would also be able to reuse `getCellSetsHierarchy` here (and check if in any other places too)